src/engine/cuecontrol.cpp

```diff
@@ -711,7 +712,7 @@ void CueControl::cueCDJ(double v) {
     // If play is pressed while holding cue, the deck is now playing. (Handled in playFromCuePreview().)
 
     QMutexLocker lock(&m_mutex);
-    bool playing = m_pPlay->toBool();
+    bool playing = m_pPlay->toBool() && !getEngineBuffer()->getScratching();
```

Comment:
What are you referring to by "opposite" here? If the track is paused `m_play` is false as it was before... 

"Scratching" just means moving the record to prevent it from playing freely. In the context of cueing it means fine tuning the cue point via the platter. I believe this PR as it is, is the correct behavior for Pioneer CDJs.

Furthermore, in the case of a very sensitive platter (like mine), there may be some non-zero transport when you're touching the platter holding the position for the queue point.